python/dgl/graph_index.py

```diff
@@ -1003,7 +996,8 @@ def from_coo(num_nodes, src, dst, is_multigraph, readonly):
     dst : Tensor
         Dst end nodes of the edges.
     is_multigraph : bool or None
-        True if the graph is a multigraph. None means determined by data.
+        Deprecated (Will be deleted in the future).
+        True if the graph is a multigraph. None will be set to True by default.
```

Comment:
We can safely delete all the `is_multigraph` argument in this file. GraphIndex is not meant for the end users. This could make things easier since you don't need to write all these deprecation messages.

Reply:
If graph_index is not exposed to user. we can delete the is_multigraph directly.